src/load.jl

```diff
@@ -94,9 +98,7 @@ end
 
 Internal function to get subject information from dataframe.
 """
-
-# get subject and file information
-function get_info!(files_df, file)
+ function get_info!(files_df, file)
```

Comment:
maybe rename to better fit it's purpose?


`get/extract_subjectid` or something?